v23.1/create-changefeed.md

```diff
@@ -351,6 +353,17 @@ CREATE CHANGEFEED FOR TABLE name, name2, name3
 
 For step-by-step guidance on creating a changefeed connected to a webhook sink, see the [Changefeed Examples](changefeed-examples.html#create-a-changefeed-connected-to-a-webhook-sink) page. The parameters table on the [Changefeed Sinks](changefeed-sinks.html#webhook-parameters) page provides a list of the available webhook parameters.
 
+### Define a key to determine the changefeed sink partition
+
+{% include_cached new-in.html version="v23.1" %} With the [`key_column`](#key-column) option, you can define the key used in message metadata that determines the partition for the changefeed message at your [downstream sink](changefeed-sinks.html). This option overrides the default primary key:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+CREATE CHANGEFEED FOR TABLE name INTO 'external://kafka-sink' WITH key_column='partition_column', unordered;
+~~~
+
+`key_column` does not preserve ordering of messages from CockroachDB to the downstream sink, therefore you **must** include the [`unordered`](#unordered) option. It does not affect per-key [ordering guarantees](changefeed-messages.html#ordering-guarantees) or the output of [`key_in_value`](#key-in-value).
```

Comment:
Does this warrant a callout?

Reply:
I have not added a callout here. Namely because if a user were to run `key_column` without `unordered` they receive a helpful warning/error, that is, they can't run without it.